public_html/index.php

```diff
@@ -118,18 +118,14 @@
             <div class="card text-white bg-dark border-light">
               <div class="card-body">
                 <pre class="text-white mb-0">
-
-<span class="text-white-50"># Prerequisites</span>
-<span class="text-white-50">* Java 7 or 8 </span>
-<span class="text-white-50">* Docker engine 1.10.x (or higher)</span>
-
+<span class="text-white-50"># Prerequisites: Java 7 or 8, Docker</span>
 <span class="text-white-50"># Install nextflow</span>
 curl -s https://get.nextflow.io | bash
+mv nextflow ~/bin
 
-<span class="text-white-50"># move nextflow file to your 'PATH'</span>
 <span class="text-white-50"># Launch the RNAseq pipeline</span>
 nextflow run nf-core/RNAseq \
-    -with-docker \
```

Comment:
why is "-with-docker" not correct here?

Reply:
With most of the nf-core pipelines, the two are functionally identical. We used to recommend using `-with-docker` and `-with-singularity` to switch these on, but when conda support was added there was no `-with-conda` flag (the option exists, but you have to point it to a specific file if I remember correctly).

After discussing this quite a bit, Paolo recommended a best practice method of using config profiles for these switches instead: https://www.nextflow.io/docs/latest/conda.html#best-practices

So now, we have `-profile [docker|singularity|conda]` instead. This aims to make the interface more consistent. Using the `-with-docker` flags should still work though.

Bit of a mess. Make sense?

Reply:
yup :)